package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Official Solium Plugin for Security-related lint rules",
   "main": "index.js",
   "scripts": {
-    "test": "rm -rf ./node_modules/solium-plugin-security && mocha --require should --reporter spec --recursive",
+    "test": "mocha --require should --reporter spec --recursive",
```

Comment:
In order to get this to green I had to remove the rm. My npm knowledge is week, so I'm not sure if this would have an unintended consequence.

Reply:
This is exactly what was needed! I couldn't figure this out on my own, so pretty sure you know more than me about npm :P